src/Hosting/TestHost/ref/Microsoft.AspNetCore.TestHost.netcoreapp3.0.cs

```diff
@@ -5,7 +5,7 @@ namespace Microsoft.AspNetCore.TestHost
 {
     public partial class ClientHandler : System.Net.Http.HttpMessageHandler
     {
-        public ClientHandler(Microsoft.AspNetCore.Http.PathString pathBase, Microsoft.AspNetCore.Hosting.Server.IHttpApplication<Microsoft.AspNetCore.Hosting.Internal.HostingApplication.Context> application) { }
+        internal ClientHandler() { }
```

Comment:
Why is this still listed in the ref assembly?

Reply:
I was curious as well... especially since the constructor parameters doesn't even match. But that's what the tool did.

Reply:
It seems to do it for any non-static type without a public constructor. That makes sense, the default constructor is public unless otherwise stated.